docs/api/public_php_api_search.md

```diff
@@ -337,6 +340,7 @@ For more information and examples, see [PagerFanta documentation.](https://githu
 |[`LocationSearchHitAdapter`](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/eZ/Publish/Core/Pagination/Pagerfanta/LocationSearchHitAdapter.php)|Makes a Location search against passed Query and  returns [SearchHit](https://github.com/ezsystems/ezplatform-kernel/blob/v1.0.0/eZ/Publish/API/Repository/Values/Content/Search/SearchHit.php) objects instead.|
 |[`ContentFilteringAdapter`](https://github.com/ezsystems/ezplatform-kernel/blob/master/eZ/Publish/Core/Pagination/Pagerfanta/ContentFilteringAdapter.php)|Applies a Content filter and returns a `ContentList` object.|
 |[`LocationFilteringAdapter`](https://github.com/ezsystems/ezplatform-kernel/blob/master/eZ/Publish/Core/Pagination/Pagerfanta/LocationFilteringAdapter.php)|Applies a Location filter and returns a `LocationList` object.|
+|[`SearchResultAdapter`](https://github.com/ezsystems/ezplatform-kernel/blob/master/eZ/Publish/Core/Pagination/Pagerfanta/SearchResultAdapter.php)|Exposes additional search result data: aggregation results, max. score, computation time, and timeout flag.|
```

Comment:
This is just an interface implemented by `ContentSearchAdapter`, `ContentSearchHitAdapter`, `` `LocationSearchAdapter` ``, `LocationSearchHitAdapter` adapters. 

In practice this mean that only those could be used with `\eZ\Publish\Core\Pagination\Pagerfanta\Pagerfanta` and only those expose addition data. Ref. https://github.com/ezsystems/ezplatform-kernel/blob/243cdae6a8b788541c7637fe0b44b82b1ea0aea9/eZ/Publish/Core/Pagination/Pagerfanta/Pagerfanta.php#L16-L19

Reply:
So do I understand correctly that it makes to sense to list it in the table?

Reply:
Yes